script.js

```diff
@@ -0,0 +1,54 @@
+const speakers = [{
+  photo: 'images/portrait1.png',
```

Comment:
- [ ] Please make sure to resize the images in the speaker section, they are too big compared to those on the template.
### Your images 👇️
![Screenshot from 2022-10-21 01-10-53](https://user-images.githubusercontent.com/89228743/197088413-2fdd72b1-98f6-460d-b876-f3770aa1a864.png)
### On the template  👇️
![Screenshot from 2022-10-21 01-15-06](https://user-images.githubusercontent.com/89228743/197088532-a424aac4-9272-4396-a1dc-2be966aa7e23.png)